R/write.R

```diff
@@ -782,6 +782,12 @@ write_data_table <- function(
   if (is.null(x)) {
     return(wb)
   }
+  # overwrite na.strings if nothing was provided
+  # with whatever is in the option if not set to default
+  if (identical(as.character(na.strings), "na_string") &&
```

Comment:
I don’t recall at the moment if we could check for inherits waiver.

Reply:
there is `is_na_strings()`, but I am somewhat scared of it.